waspc/data/Generator/templates/sdk/wasp/auth/utils.ts

```diff
@@ -155,12 +161,6 @@ export async function deleteUserByAuthId(authId: string): Promise<{ count: numbe
   } } })
 }
 
-export async function createAuthToken(
```

Comment:
This guy is missing, that's also correct?

Reply:
We are no longer using JWTs for our auth, but rather the session ID that's stored in the DB :)